source/deprecations/v2.x.html.md

```diff
@@ -525,3 +525,52 @@ Once view and controller deprecations are removed, you can remove the addons wit
 
 `Ember.Backburner` was private throughout the Ember 2.x series and will be
 removed _after_ 2.8.
+
+#### Route#serialize
+
+##### until: 3.0.0
+##### id: ember-routing.serialize-function
+
+The `Route#serialize` function was deprecated in favor of passing a `serialize` function into the route's definition in the router map.
```

Comment:
Can you link to the finalized/merged RFC (not the PR, but the template created)?